backend/services/chatClient.js

```
@@ -480,84 +480,104 @@ class ChatClient {
-            iconUrl: 'https://fonts.gstatic.com/s/i/materialicons/schedule/v1/24px.svg'
-          }
+      default:
+        // Enhanced detailed format for other events
+        let action = 'updated';
+        if (event.includes('create')) action = 'created';
+        if (event.includes('add')) action = 'added';
+        if (event.includes('delete')) action = 'deleted';
+
+        message = `🔔 *${userName}* ${action} ${objectType}: *${objectName}*\n\n`;
+        if (value) message += `💰 Value: *${value}*\n`;
+        if (object?.stage_id) message += `🎯 Stage: ${object.stage_id}\n`;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)